### PR TITLE
[SN-120] Include examples of using composite mask in image projects

### PR DESCRIPTION
**Pull request by ovalle15**
- Included examples of uploading composite masks in image projects. To test please create an import with both Annotation types and NDJSON
-  Removed how to fetch a single instance mask since this will be deprecated, and for composite mask we have a different notebook demonstrating how to do this

### CODE REVIEW

**Comment by Gabefire** [comment]
image
Hey! I know this was not part of your PR but for the global_key on step 1 of the end to end. I think it would be better to do a `str(uuid.uuid4())` instead of the image name. I often run into global key collisions because other notebooks uses this global key plus if I run the notebook again it would error out at that point. Let me know what you think!

File: examples/annotation_import/image.ipynb [hunk 1]
```diff
@@ -500,47 +502,27 @@
     {
       "metadata": {},
       "source": [
-        "### Segmentation Mask"
+        "### Composite mask upload using multiple tools \n",
```
[the remark this answers is not in the document]
Comment (ovalle15) [reply examples/annotation_import/image.ipynb:505]: How about : 
"Composite mask upload using different tools "
I am demonstrating how you can match colors to tools, so keeping the word tool is essential.

**Comment by ovalle15** [comment]
re: https://github.com/Labelbox/labelbox-python/pull/1431#issuecomment-1964624796 
I think this is a feature, not necessarily an issue because it demonstrates the whole point of using global keys, which is avoiding uploading duplication of data. 
The user can still use the data that exists in their organization without issues in the notebook

File: examples/annotation_import/image.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gabefire) [reply examples/annotation_import/image.ipynb:505]: Yea I know I just think of tools as in something more broad like an entire library/ module (git for example) were function/ method would be what this is in my mind. We can keep it though if you think it is essential and it is minor

**Comment by ovalle15** [comment]
I totally get it ! I made some updates incorporated your suggestions ! @Gabefire

**Review by Gabefire — APPROVED** [review]

**Comment by Gabefire** [comment]
Looks good!

**Pull request merged**